proto/marshal/compat.py

```diff
@@ -44,16 +45,16 @@
 # See https://github.com/protocolbuffers/protobuf/issues/16596
 map_composite_type_names = ("MessageMapContainer",)
 
-if _message:
-    repeated_composite_types += (_message.RepeatedCompositeContainer,)
-    repeated_scalar_types += (_message.RepeatedScalarContainer,)
-
-    try:
-        map_composite_types += (_message.MessageMapContainer,)
-    except AttributeError:
-        # The `MessageMapContainer` attribute is not available in Protobuf 5.x+
-        pass
+for message in [_message_upb, _message_pyext]:
```

Comment:
Why do we need a for loop here? Are `_message_upb` and `_message_pyext` mutually exclusive?

Reply:
As per the comment at line 23, we want to include types from  both `_message_upb` and `_message_pyext`